1559771746894	Sync.LogManager	DEBUG	Flushing file log
1559771746924	Sync.LogManager	DEBUG	Log cleanup threshold time: 1558907746924
1559771747051	Sync.LogManager	DEBUG	Done deleting files.
1559773670234	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559773670319	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559773670319	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559773670321	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559773670321	Sync.Service	INFO	Starting sync at 2019-06-06 00:27:50 in browser session DLDB2kz0MYXZ
1559773670322	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1559773670324	Sync.Status	INFO	Resetting Status.
1559773670324	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1559773670757	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559773670930	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1559773670930	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1559773670930	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1559773671692	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559773672194	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559773672194	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559773672196	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559773672196	Sync.Service	INFO	Starting sync at 2019-06-06 00:27:52 in browser session DLDB2kz0MYXZ
1559773672199	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:41:5) JS Stack trace: LockException@util.js:41:5
throwLockException@util.js:115:11
WrappedLock@util.js:129:15
_lockedSync@service.js:1164:8
sync/<@service.js:1118:18
WrappedCatch@util.js:103:27
sync@service.js:1119:7
1559773672200	Sync.Service	INFO	Cannot start sync: already syncing?
1559773673149	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559775590152	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559775590281	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559775590281	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559775590282	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559775590282	Sync.Service	INFO	Starting sync at 2019-06-06 00:59:50 in browser session DLDB2kz0MYXZ
1559775590282	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:41:5) JS Stack trace: LockException@util.js:41:5
throwLockException@util.js:115:11
WrappedLock@util.js:129:15
_lockedSync@service.js:1164:8
sync/<@service.js:1118:18
WrappedCatch@util.js:103:27
sync@service.js:1119:7
1559775590282	Sync.Service	INFO	Cannot start sync: already syncing?
1559775590814	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559775591163	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1559775591163	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1559775591164	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1559775591695	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559775591756	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559775592025	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559775592025	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559775592027	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559775592027	Sync.Service	INFO	Starting sync at 2019-06-06 00:59:52 in browser session DLDB2kz0MYXZ
1559775592028	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:41:5) JS Stack trace: LockException@util.js:41:5
throwLockException@util.js:115:11
WrappedLock@util.js:129:15
_lockedSync@service.js:1164:8
sync/<@service.js:1118:18
WrappedCatch@util.js:103:27
sync@service.js:1119:7
1559775592028	Sync.Service	INFO	Cannot start sync: already syncing?
1559775606236	Sync.Resource	WARN	GET request to https://sync-535-us-west-2.sync.services.mozilla.com/1.5/80856074/info/collections failed: [Exception... "Failure"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "<unknown>"  data: no] No traceback available
1559775606274	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "Failure"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "<unknown>"  data: no] No traceback available
1559775606379	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1559775606831	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "Failure"  nsresult: "0x80004005 (NS_ERROR_FAILURE)"  location: "<unknown>"  data: no] No traceback available
